src/client/options/mod.rs

```diff
@@ -575,6 +588,12 @@ pub(crate) struct TestOptions {
 
     /// Mock response for `SrvPollingMonitor::lookup_hosts`.
     pub(crate) mock_lookup_hosts: Option<Result<LookupHosts>>,
+
+    /// Optionally stores the unified test runner entity ID for this client.
+    /// Used to include client IDs in tracing events which allows correlating
+    /// the events with specific clients.
+    #[cfg(feature = "tracing-unstable")]
+    pub(crate) client_id: Option<String>,
```

Comment:
>So this property is actually on TestOptions, which is only defined under #[cfg(test)]. so I'm not sure it makes sense to add that comment here given this definition is not included in the driver. (the way GH collapses the intermediate code does make it look like this is on ClientOptions though lol)

Ah ok, I missed that. I thought this was on `ClientOptions`. No need for a comment here.

>that said, is there any nice way to avoid the unconditional definition there?

Hmm I don't think so, or at least I don't know of any way around it. You could look into the generated code of the macro and possibly use it there, but I doubt that will end up being cleaner overall.